pb/summary/summary.proto

```diff
@@ -45,6 +47,61 @@ message FailingTestSummary {
   string fail_test_link = 13;
 }
 
+// Metrics about a specific test, i.e. passes, fails, total runs, etc.
+message TestInfo {
+  // The display name of the test, typically what is shown on the leftmost 
+  // column of each row of TestGrid
```

Comment:
nit: "typically what is shown for each row in TestGrid" might read more clearly?

Reply:
Done